### DISCUSSION TITLE
:beer:  v0.8.1 not (yet) available on brew :beer:

### DISCUSSION THREAD

**Discussion by adriens** [duckdb/duckdb#7929] [Ideas]
[`v0.8.1`](https://github.com/duckdb/duckdb/releases/tag/v0.8.1) has been released but while trying to upgrade the latest release is not available : 

![image](https://github.com/duckdb/duckdb/assets/5235127/50e2098b-cc36-4d74-89a3-f869325549bc)

**Comment by Mause** [comment]
Updating brew is not automatic, you need to wait for this pr to be merged: https://github.com/Homebrew/homebrew-core/pull/133489

**Reply by adriens** [reply]
Thanks a lot @Mause :pray:

**Comment by adriens** [comment]
:clap: 
![image](https://github.com/duckdb/duckdb/assets/5235127/019f6368-7ddd-43e4-add5-978a431fdef9)
